README.md

```diff
@@ -1,21 +1,51 @@
 # React Updating State Lab
 
 ## Objectives
-
 1. Practice updating state using `setState()`
-2. Explain the difference between a React SyntheticEvent and a DOM Event
-3. Practice using controlled form components
 
-## Overview
+## World record click counter
+![Oldtimer stuff](http://il5.picdn.net/shutterstock/videos/15633112/thumb/1.jpg)
 
-As usual, we want students to practice what the reading has covered. We can
-start homing in on the difference between props and state here, as it will
-be useful to them pretty much always going forward.
+We got a new job! Woo! This time, we're a judge for Guinness World Records. We're no ordinary judge, however — we're a
+_modern day_ judge. We do things using _computers_. Instead of using those old, clunky clickity-things to count stuff
+for world records, we'll write our very digital clicker!
 
-The components students build in this lab should be a bit larger than in
-previous labs, but not gargantuan yet.
+1. In the `components/DigitalClicker.js` file, create a `DigitalClicker` React component.
+2. This component has a state property called `timesClicked`, that is initially defined as 0.
+3. The component renders out a button with a label that shows the `timesClicked` value. This means that at the start,
+your button should just say `0`.
+4. Whenever the button is clicked, update the state by incrementing the `timesClicked` by 1.
 
-## Resources
 
-- [Value of `null` for Controlled Input](https://facebook.github.io/react/tips/controlled-input-null-value.html)
-- [React: Forms](https://facebook.github.io/react/docs/forms.html)
+## Debugging YouTube
+![Spock](https://media.giphy.com/media/fECTyvPYevOHC/giphy.gif)
+
+You can only count so many jumps, hula-hoop rotations and hot dogs being devoured before you go insane. Time to move on
+from our record judge job to greener pastures! Not to worry, we've secured ourselves a job at YouTube this time! Our
+first job is to create some sort of debug bar that allows us to control the simulated network conditions on YouTube.
```

Comment:
minor suggestion: maybe: "our first **task**" instead of "our first job", just to avoid repetition.